### DISCUSSION TITLE
Error in using FragmentHistogram()

### DISCUSSION THREAD

**Discussion by rainion2016** [stuart-lab/signac#1208] [Q&A] [answered]
Hi @timoast, thanks for the nice package. Recently I am trying to use FragmentHistogram() and am encountering an error. While I kind of remember that I could sucessfully plot the histogram months ago. I wonder if it is caused by the version update and somehow it's not compatible with ggplot2.

Here's the error
`> pbmc <- readRDS("pbmc.Rds")`
`> pbmc$pct_reads_in_peaks <- pbmc$peak_region_fragments / pbmc$passed_filters * 100`
`> pbmc$blacklist_ratio <- pbmc$blacklist_region_fragments / pbmc$peak_region_fragments`
`> pbmc$nucleosome_group <- ifelse(pbmc$nucleosome_signal > 4, 'NS > 4', 'NS < 4')`
`> FragmentHistogram(object = pbmc, group.by = 'nucleosome_group')`

> Error in `combine_vars()`:
! Faceting variables must have at least one value
Run `rlang::last_error()` to see where the error occurred.

`> rlang::last_error()`

> Error in `combine_vars()`:
! Faceting variables must have at least one value
Backtrace:
   1. base (local) `<fn>`(x)
   2. ggplot2:::print.ggplot(x)
   4. ggplot2:::ggplot_build.ggplot(x)
   5. layout$setup(data, plot$data, plot$plot_env)
   6. ggplot2 (local) f(..., self = self)
   7. self$facet$compute_layout(data, self$facet_params)
   8. ggplot2 (local) f(...)
 
  **11. ggplot2::combine_vars(data, params$plot_env, vars, drop = params$drop)**

Run `rlang::last_trace()` to see the full context.

Version information:
Signac 1.7.0
ggplot2 3.3.6
base 4.1.3
rlang 1.0.5
Thank you!

**Comment by timoast** [comment, marked as answer]
Hi, it's likely that you don't have any cells with nucleosome_signal < 4 here, and so there's no cells to plot in one of the groups. The steps shown in the vignette are only meant as an example of how to use functions in the package, you'll need to adjust parameters according to your dataset.

Please use the github issues for errors and bug reports

**Reply by rainion2016** [reply]
Thanks, I reviewed the data of pbmc$nucleosome_signal and noticed something weird.  It seemed to have been log normalized/scaled.   I just modified it and it finally worked.

**Reply by xiaoxiaoh16** [reply]
Hi, I have same issue here, could you please explain  a little more about how modify it? thank you in adance!
